deploy/stacks/pipeline.py

```diff
@@ -206,9 +206,6 @@ def __init__(
             else:
                 self.set_albfront_stage(target_env, repository_name)
 
-        if self.node.try_get_context('git_release'):
-            self.set_release_stage()
```

Comment:
Can we also delete `self.git_project_role()` and associated policies?

Reply:
That is already done right?